docs/src/man/man1/mitsub_vfd.1.adoc

```diff
@@ -9,148 +9,129 @@ connector.
+_VFD_NAME_**.monitor** (bit, in):: Set monitor mode pin.
+If false, request-status command will not be sent to VFD.
+Status, amps, power, motor-feedback, and alarm would then not be useful.
+_VFD_NAME_**.estop** (bit, in):: Set E-stop mode pin.
 This will stop the VFD. Restarting requires the run pin to cycle.
-
-*[VFD NAME].fwd* (bit, out):: up-to-speed status pin Motor is at
-requested speed within VFD's settings tolerance. +
-*[VFD NAME].alarm* (bit, out):: alarm status pin +
-
-*[VFD NAME].motor-cmd* (float, in):: The requested motor speed, is
-Hertz +
-
-*[VFD NAME].motor-fb* (float, out):: The motor feedback speed (from VFD)
-in Hertz (Hz) +
-*[VFD NAME].motor-amps* (float, out):: The motor current, in amps +
-*[VFD NAME].motor-power* (float, out):: The motor power +
-
-*[VFD NAME].scale-cmd* (float, in):: Motor command's scale setting
-defaults to 1 +
-*[VFD NAME].scale-cmd* (float, in):: Motor command's scale setting
-defaults to 1 +
-*[VFD NAME].scale-cmd* (float, in):: Motor command's scale setting
-defaults to 1 +
-
-*[VFD NAME].stat-bit-0* (bit, out):: raw status bit +
-*[VFD NAME].stat-bit-1* (bit, out):: raw status bit +
-*[VFD NAME].stat-bit-2* (bit, out):: raw status bit +
-*[VFD NAME].stat-bit-3* (bit, out):: raw status bit set the VFD so this
-is motor-at-speed status +
-*[VFD NAME].stat-bit-4* (bit, out):: raw status bit +
-*[VFD NAME].stat-bit-5* (bit, out):: raw status bit +
-*[VFD NAME].stat-bit-6* (bit, out):: raw status bit +
-*[VFD NAME].stat-bit-7* (bit, out):: raw status bit Set the VFD so this
-in the alarm bit
+_VFD_NAME_**.fwd** (bit, out):: Up-to-speed status pin Motor is at requested speed within VFD's settings tolerance.
+_VFD_NAME_**.alarm** (bit, out):: Alarm status pin
+_VFD_NAME_**.motor-cmd** (float, in):: The requested motor speed, in Hertz (Hz)
+_VFD_NAME_**.motor-fb** (float, out):: The motor feedback speed (from VFD) in Hertz (Hz)
+_VFD_NAME_**.motor-amps** (float, out):: The motor current, in amperes (A)
+_VFD_NAME_**.motor-power** (float, out):: The motor power
+_VFD_NAME_**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
+_VFD_NAME_**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
+_VFD_NAME_**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
+_VFD_NAME_**.stat-bit-0** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-1** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-2** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-3** (bit, out):: Raw status bit. Set the VFD so this is motor-at-speed status. FIXME: Unclear - is the signal set or is the wiring to be set in a particular way?
+_VFD_NAME_**.stat-bit-4** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-5** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-6** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-7** (bit, out):: Raw status bit. Set the VFD so this in the alarm bit. FIXME: Unclear - is the signal set or is the wiring to be set in a particular way?
```

Comment:
```suggestion
_VFD_NAME_**.stat-bit-7** (bit, out):: Raw status bit. Configure the VFD so that the function 'alarm' is assigned to this status bit (parameter 195 for 700 series)
```